data/saas/config/saas_example_config.yml

```diff
@@ -70,15 +70,24 @@ saas_config:
     - name: member
       requests:
         read:
-          method: GET
-          path: /3.0/search-members
-          query_params:
-            - name: query
-              value: <email>
-          param_values:
-            - name: email
-              identity: email
-          data_path: exact_matches.members
+          - method: GET
```

Comment:
The config changed to be able to define a _list_ of `read` requests. We use the provided identity to determine which read request to use, or return all read requests if none match the current identity (this is the base case since most read requests will just specify one request that doesn't rely on the identity param value).